### PR TITLE
System comments

### PR DESCRIPTION
**Pull request by dgarcia360**
Added a reference and fixed a typographical convention. 

Other comments not solved in this PR:

- [x] Handlers: These are APIs that are always accessible. -> By whom? You could also add an example of a handler.
- [x] Diagnostics: I don't understand the meaning of the word ``counters`` in that context.
- [x] Broker: "It’s expected that the server and broker have different extensions loaded."→ Why brokers and servers of the same network are expected to have different extensions loaded?

### CODE REVIEW

**Comment by Jaguar0625** [comment]
> Handlers: These are APIs that are always accessible. -> By whom? You could also add an example of a handler.

All clients. There is no filter.

>  Diagnostics: I don't understand the meaning of the word counters in that context.

think of it like a win32 performance counter. for example, we have counters like:
```
   ACCTREST C : 0     |      ACNTST C : 33    |  ACNTST C HVA : 9     |  ACTIVE PINGS : 0     | 
    B WRITERS : 0     |       BAN ACT : 0     |       BAN ALL : 0     |  BLK ELEM ACT : 0     | 
 BLK ELEM TOT : 182   |      BLKDIF C : 41    |        HASH C : 39    |    HASHLOCK C : 0     | 
  MEM CUR RSS : 85    |  MEM CUR VIRT : 4'758 |   MEM MAX RSS : 87    |   MEM SHR RSS : 37    | 
   METADATA C : 0     |      MOSAIC C : 2     |  MOSAICREST C : 0     |    MULTISIG C : 0     | 
        NODES : 8     |          NS C : 1     |       NS C AS : 3     |       NS C DS : 3     | 
RB COMMIT ALL : 2     | RB COMMIT RCT : 1     | RB IGNORE ALL : 4     | RB IGNORE RCT : 4     | 
      READERS : 8     |  SECRETLOCK C : 0     | SUCCESS PINGS : 32    |         TASKS : 13    | 
TOT CONF TXES : 39    |   TOTAL PINGS : 32    |   TS NODE AGE : 2     | TS OFFSET ABS : 0     | 
TS OFFSET DIR : 0     |  TS TOTAL REQ : 8     |   TX ELEM ACT : 0     |   TX ELEM TOT : 0     | 
 UNLKED ACCTS : 1     |      UT CACHE : 0     |       WRITERS : 5     |  
```

>  Broker: "It’s expected that the server and broker have different extensions loaded."→ Why brokers and servers of the same network are expected to have different extensions loaded?

Consider peer, api and dual server. They will all have different extensions loaded.
Additionally, some api/dual nodes might want to support zmq but not mongo or vice versa.

**Comment by dgarcia360** [comment]
@Jaguar0625 Thanks for the explanations! I just submitted some new suggestions.

File: whitepaper/chapters/system.tex [hunk 1]
```diff
@@ -38,28 +38,28 @@ \subsection{Transaction Plugins}
 
 \begin{enumerate}
 	\item{Transactions -
-		New transaction types and the mapping of those types to parsing rules can be specified.
+		Specify new transaction types and the mapping of those types to parsing rules.
```
Comment (Jaguar0625) [inline whitepaper/chapters/system.tex:41]: Specify or Define ?

File: whitepaper/chapters/system.tex [hunk 2]
```diff
@@ -38,28 +38,28 @@ \subsection{Transaction Plugins}
...
-	\item{Handlers - These are APIs that are always accessible.}
-	\item{Diagnostics - These are APIs and counters that are accessible only when the node is running in diagnostic mode.}
-	\item{Validators -
-		Stateless and stateful validators process the notifications produced by block and transaction processing.
-		The registered validators can subscribe to general or plugin-defined notifications and reject disallowed values or state changes.
+	\item{Handlers - Define public APIs that are always accessible.}
+	\item{Diagnostics - Define APIs and counters that are accessible only when the node is running in diagnostic mode.}
+	\item{Validators - Process the notifications produced by block and transaction processing.
+			These validators can be stateless or stateful.
+			The registered validators can subscribe to general or plugin-defined notifications and reject disallowed values or state changes.
```
Comment (Jaguar0625) [inline whitepaper/chapters/system.tex:53]: extra indent

File: whitepaper/chapters/system.tex [hunk 3]
```diff
@@ -38,28 +38,28 @@ \subsection{Transaction Plugins}
...
+	\item{Observers - Process the notifications produced by block and transaction processing.
 		The registered observers can subscribe to general or plugin-defined notifications and update blockchain state based on their values.
 		Observers don't require any validation logic because they are only called after all applicable validators succeed.
 	}
 	\item{Resolvers -
-		Custom mappings from unresolved to resolved types can be specified.
-		For example, this is used by the namespace plugin to add support for alias resolutions.
+		Specify custom mappings from unresolved to resolved types.
+		For example, this is used by the namespace plugin to add support for alias resolutions
+\nemrefparens{sec:accounts:addressaliases}.
```
Comment (Jaguar0625) [inline whitepaper/chapters/system.tex:62]: indent or move to prev line

File: whitepaper/chapters/system.tex [hunk 4]
```diff
@@ -176,6 +175,7 @@ \subsection{Broker}
 For example, overriding the network time supplier in the broker is not supported.
 Broker extensions are primarily intended to register subscribers and react to events forwarded to those subscribers.
 Accordingly, it's expected that the server and broker have different extensions loaded.
+To illustrate, the server might want to support harvesting capabilities, but the broker does not normally load this extension.
```
Comment (Jaguar0625) [inline whitepaper/chapters/system.tex:178]: ~capabilities~ ? (not sure)

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]

**Comment by dgarcia360** [comment]
@Jaguar0625 Do not merge this one yet. I'll review the consistency with bullets points from other sections (if they start with verb or noun).

**Comment by dgarcia360** [comment]
If we want to apply those changes we would need to read every other bullet point. Closing this for now, if necessary will open a new PR.

**Pull request closed without merging**